src/v2/api/controllers/GroupController.ts

```diff
@@ -122,54 +141,118 @@ export class GroupController {
+      InvalidEntityIdException: Group with such id is not found
+    `,
+  })
+  @ApiForbiddenResponse({
+    description: `
+      NoPermissionException: You do not have permission to perform this action
+    `,
+  })
   @Access('groups.$groupId.students.get')
   @Get('/:groupId/students')
-  async getStudents (
-    @Param('groupId', GroupByIdPipe) groupId: string,
-  ) {
+  async getStudents(@Param('groupId', GroupByIdPipe) groupId: string) {
     const students = await this.groupService.getStudents(groupId);
 
     return { students };
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: CaptainResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidEntityIdException: Group with such id is not found
+    `,
+  })
+  @ApiForbiddenResponse({
+    description: `
+      NoPermissionException: You do not have permission to perform this action
+    `,
+  })
   @Access('groups.$groupId.captain.get')
   @Get('/:groupId/captain')
-  async getCaptain (
-    @Param('groupId', GroupByIdPipe) groupId: string,
-  ) {
+  async getCaptain(@Param('groupId', GroupByIdPipe) groupId: string) {
     const captain = await this.groupService.getCaptain(groupId);
     if (!captain) {
       throw new AbsenceOfCaptainException();
     }
     return captain;
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: [DisciplineTeachersResponse],
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidEntityIdException: Group with such id is not found
+    `,
+  })
+  @ApiForbiddenResponse({
+    description: `
+      NoPermissionException: You do not have permission to perform this action
+    `,
+  })
   @Access('groups.$groupId.disciplines.teachers.get')
   @Get('/:groupId/disciplineTeachers')
-  async getDisciplineTeachers (
-    @Param('groupId', GroupByIdPipe) groupId: string,
+  async getDisciplineTeachers(
+    @Param('groupId', GroupByIdPipe) groupId: string
   ) {
     return this.groupService.getDisciplineTeachers(groupId);
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: [ShortDisciplineResponse],
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidEntityIdException: Group with such id is not found
+    `,
+  })
+  @ApiForbiddenResponse({
+    description: `
+      NoPermissionException: You do not have permission to perform this action
+    `,
+  })
   @Access('groups.$groupId.disciplines.get')
   @Get('/:groupId/disciplines')
-  async getDiscipline (
-    @Param('groupId', GroupByIdPipe) groupId: string,
-  ) {
+  async getDiscipline(@Param('groupId', GroupByIdPipe) groupId: string) {
     const disciplines = await this.groupService.getDisciplines(groupId);
     return { disciplines };
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: ShortUsersResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidEntityIdException: Group with such id is not found
+      InvalidBodyException: Email can not be empty
+      InvalidBodyException: One of emails is not an email
```

Comment:
Change this description to : `The email is not a valid email address`

Reply:
Also on line 244, change the description to: `Email cannot be empty`

Reply:
"InvalidEntityIdException" must be "InvalidGroupIdException"